src/main/java/nl/workingtalent/bookrental/model/Book.java

```diff
@@ -14,6 +15,14 @@
 @Entity
 public class Book {
 
+	/*
+	 * public Book(String title, String author, String isbn) { super(); this.title =
+	 * title; this.author = author; this.isbn = isbn;
+	 * 
+	 * this.tags = new ArrayList<Tag>();; this.copies = new ArrayList<Copy>();
+	 * this.reservations = new ArrayList<Reservation>();; }
+	 */
+
```

Comment:
? Dit lijkt gewoon te kloppen?

Reply:
Ik zou dit soort dingen/comments niet in de code laten staan